p/themes/Origine/origine.css

```diff
@@ -743,14 +743,14 @@ a.btn {
 	background: #0084cc;
 	text-align: center;
 	font-size: 0.9em;
+	padding: 1em;
```

Comment:
Is the order right?

Reply:
is there a rule of ordering the css rules?

Reply:
Yes, but there seems to be some issue with the CI atm.

Reply:
https://github.com/FreshRSS/FreshRSS/blob/edge/.stylelintrc

Reply:
Should go at the top, not bottom.

Reply:
fixed